manifests/params.pp

```diff
@@ -29,11 +29,13 @@
       # Redhat/Centos >= 7.0
       if(versioncmp($::operatingsystemrelease, '7.0') >= 0) and $::operatingsystem != 'Amazon' {
         $additional_packages = ['easy-rsa']
-        $ldap_auth_plugin_location = undef
+        $ldap_auth_plugin_package = ['openvpn-auth-ldap']
```

Comment:
does it need to be a an array or should we stick with a string? I think string is enough.